builtin.go

```diff
@@ -622,6 +622,14 @@ var Panic = &Function{
 	FixedArgs: 1,
 }
 
+var Exit = &Function{
```

Comment:
this is exactly what I wanted to add, and will use also instead of the current panics! :heart: